### ISSUE TITLE
Separate alignwide and alignfull theme support

### ISSUE THREAD

**Issue by mrwweb** [WordPress/gutenberg#14133]
**Is your feature request related to a problem? Please describe.**
I was quite confused at first that `add_theme_support( 'align-wide' )` adds theme support for BOTH `alignwide` and `alignfull` classes.

I would expect it be possible that a theme could support one without the other for stylistic reasons.

**Describe the solution you'd like**
`add_theme_support( 'align-full' )`

OR

`add_theme_support( 'align-full', false )` if required for backwards compat

**Comment by strarsis** [comment]
Yes, a `wide` or `full` option may not make sense in all themes, some themes may only want to offer a `full` alignment.

**Comment by addzycullen** [comment]
I +1 this, seems silly that they are both controlled in the same statement.

This needs to be separated

**Comment by youknowriad** [comment]
Closing as resolved by theme.json: alignment availability is now governed by layout settings (contentSize/wideSize), giving themes the granular control this asked for — the monolithic align-wide theme-support flag is the legacy path. If a specific combination (e.g., full-only) still can't be expressed, a fresh scoped issue would be great. Thanks!
